src/styles/blocks/body.scss

```diff
@@ -0,0 +1,36 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+body {
+  margin: 0;
+  color: $black-color;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), colors should be stored in variables for reusability and maintainability. Consider defining `$black-color` in `variables.scss` if it has not been defined yet.